### PR TITLE
Native YAML - Web infrastructure

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
web_infrastructure/*

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

@gundalow

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @Fale. To the current maintainers, @tarka, @ramondelafuente, @privateip, @jhoekx, @oboukili please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: web_infrastructure/jira.py [hunk 1]
```diff
@@ -105,59 +105,89 @@
...
 - name: Comment on issue
-  jira: uri={{server}} username={{user}} password={{pass}}
-        issue={{issue.meta.key}} operation=comment 
-        comment="A comment added by Ansible"
+  jira:
+    uri: '{{ server }}'
+    username: '{{ user }}'
+    password: '{{ pass }}'
+    issue: '{{ issue.meta.key }}'
+    operation:comment
```
Comment (gundalow) [inline web_infrastructure/jira.py]: Missing space

File: web_infrastructure/jira.py [hunk 2]
```diff
@@ -105,59 +105,89 @@
...
+    comment:A comment added by Ansible
```
Comment (gundalow) [inline web_infrastructure/jira.py]: Missing space

File: web_infrastructure/jboss.py [hunk 3]
```diff
@@ -52,11 +52,21 @@
...
+    src: /tmp/hello-1.0-SNAPSHOT.war
+    deployment: hello.war
+    state: present
+
 # Update the hello world application
-- jboss: src=/tmp/hello-1.1-SNAPSHOT.war deployment=hello.war state=present
+- jboss:
+    src: /tmp/hello-1.1-SNAPSHOT.war
+    deployment: hello.war
+    state:present
```
Comment (gundalow) [inline web_infrastructure/jboss.py]: Missing space

**Review by gundalow — CHANGES_REQUESTED** [review]
needs_revision

I just happened to notice some missing spaces between `key:` and `value` on this PR. That might have been an issue on some of the previous PRs.

**Comment by Fale** [comment]
Thanks @gundalow, good catch :)

**Review by gundalow — APPROVED** [review]

**Pull request merged**